cf-oscore/src/main/java/org/eclipse/californium/oscore/OSCoreCtx.java

```diff
@@ -111,7 +113,18 @@
 	 * See https://tools.ietf.org/html/rfc8613#appendix-B.2
 	 */
 	private boolean contextRederivationEnabled;
-	
+
+	/**
+	 * When using outer block-wise with OSCORE a proxy can maliciously inject
+	 * block fragments. To protect against this a message with size exceeding
+	 * this parameter should never be sent without inner block-wise. Likewise
+	 * when receiving a message using outer block-wise it should be discarded if
+	 * the cumulated size exceeds this parameter.
+	 * 
+	 * See https://tools.ietf.org/html/rfc8613#section-4.1.3.4.2
+	 */
+	private int MAX_UNFRAGMENTED_SIZE;
```

Comment:
Only static final uses UPPER_CASE. Please use CaMeL, `maxUnfragmentedSize`.

Reply:
Thanks, I will change it.

Reply:
This has now been fixed.